Epsilon.Abstractions/Export/ICanvasModuleExporter.cs

```diff
@@ -1,8 +1,8 @@
-﻿using Epsilon.Canvas.Abstractions.Model;
+﻿using Epsilon.Abstractions.Model;
```

Comment:
I see inconsistent naming of plural "meervoud" naming, either we use Model**s** or Model. I don't have a preference, but we should be consistent.

Reply:
I can't find the "plural" version, Models, anywhere. So let's just keep using Model yes.